tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_component.ng.html

```diff
@@ -0,0 +1,30 @@
+<!--
+@license
+Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<div (dblclick)="typeStepRequested()">
+  <span [style.transform]="getCenteringTransform()">
+    <input
+      *ngIf="isTyping"
+      type="text"
```

Comment:
Can you actually have both type="text" and type="number"? What happens when you do that (maybe worth leaving a comment or link)?

Reply:
Actually html just takes the last value when you a have duplicates. I meant to remove text when I put number in there but, forgot as I was focused on removing the arrows. I have removed it now.